scenario/LiquidationScenario.ts

```diff
@@ -161,6 +161,37 @@ scenario(
   }
 );
 
+scenario(
+  'Comet#liquidation > user can end up with a minted supply',
+  {
+    tokenBalances: {
+      $comet: { $base: 1000 },
+    },
+    cometBalances: {
+      albert: {
+        $base: -1000,
+        $asset0: 0.001
+      },
+    },
+  },
+  async ({ comet, actors }, context, world) => {
+    const { albert, betty } = actors;
+
+    await world.increaseTime(
+      await timeUntilUnderwater({
+        comet,
+        actor: albert,
+      })
+    );
+
+    const ab0 = await betty.absorb({ absorber: betty.address, accounts: [albert.address] });
+    expect(ab0.events?.[2]?.event).to.be.equal('Transfer');
```

Comment:
Cool, lets also check that this is a `mint` event as opposed to any random ERC20 transfer event:

```
expect(event(ab0, 2)).to.be.deep.equal({
      Transfer: {
        amount: baseBalance,
        from: ethers.constants.AddressZero,
        to: albert.address,
      }
    });
```

Reply:
This doesn't handle the pre-upgrade failure cases as gracefully, with an expectation failure like we made the other change to support, I think the presence of Transfer event should actually be sufficient, although could add a check for each of the properties if you really think its important?